frontend/src/components/login.js

```diff
@@ -1,44 +1,165 @@
 import { useState } from 'react';
-import { loginUser } from '../http/user'
+import { registerUser, loginUser } from '../http/user'
 
-const Login = ({ setUserState }) => {
+const Login = ({ setUserState, setMessage, setReturnError }) => {
+    //login states
     const [userInput, setUserInput] = useState('');
     const [passwordInput, setPasswordInput] = useState('');
-    const [loginError, setLoginError] = useState('');
+    //register states
+    const [userRegInput, setUserRegInput] = useState('');
+    const [passwordRegInput, setPasswordRegInput] = useState('');
```

Comment:
We should have a confirmPassword, setConfirmPassword state as well. Normally when you sign up for accounts their's a second box that asks you to reconfirm what you just entered to ensure you didn't have any typos.

Reply:
added logic